curriculum/challenges/english/25-front-end-development/lab-profile-lookup/5688e62ea601b2482ff8422b.md

```diff
@@ -0,0 +1,132 @@
+---
+id: 5688e62ea601b2482ff8422b
+title: Build a Profile Lookup
+challengeType:  0
+dashedName: lab-profile-lookup
+---
+
+# --description--
+
+In this lab, you will build a profile lookup that looks up information about people in a contacts list.
+
+For this example imagine there is a contact named Akira Laine, the `lookUpProfile("Akira","lastName")` should return `Laine`.  
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab
+
+**User Stories:**
+
+1. You should create a function named `lookUpProfile` that takes a name and a property as arguments.
+2. If the function receives a contact name and the property exists on that contact; the property's value should be returned.
+3. If the name passed to the function does not match any contacts, then the function should return `No such contact`.
+4. If the property does not exist on a found contact, then the function should return "No such property". 
+
+# --before-all--
+
+```js
+const contacts = [
+  {
+    firstName: "Akira",
+    lastName: "Laine",
+    number: "0543236543",
+    likes: ["Pizza", "Coding", "Brownie Points"],
+  },
+  {
+    firstName: "Harry",
+    lastName: "Potter",
+    number: "0994372684",
+    likes: ["Hogwarts", "Magic", "Hagrid"],
+  },
+  {
+    firstName: "Sherlock",
+    lastName: "Holmes",
+    number: "0487345643",
+    likes: ["Intriguing Cases", "Violin"],
+  },
+  {
+    firstName: "Kristian",
+    lastName: "Vos",
+    number: "unknown",
+    likes: ["JavaScript", "Gaming", "Foxes"],
+  },
+];
+```
+
+# --hints--
+
+You should have a function named `lookUpProfile`.
+
+```js
+assert.isFunction(lookUpProfile);
+```
+
+You should have an array named `contacts`.
```

Comment:
This test is passing on seed code.

Reply:
It looks like that test doesn't need to be there at all. Because looking at the seed code we are already given them the `contacts` array. So if that is going to stay, then this test should be removed.

If we are not going to provide the contacts array for them, then the directions need to updated to tell them to create the contacts array and show them the general structure.